config/locales/en.yml

```diff
@@ -359,6 +370,7 @@ en:
       in several counties, or "Online" if it operates mostly (or solely) online.
     website_include_http: Company website
     org_nr_no_hyphens: Numbers only (no hyphens)
+    info: Company Information
```

Comment:
"Customized Company Information"  is my suggestion.  It communicates to the _member_ that this is information that they enter it and customize it (_format it_) like they want.